docs/api/rest_api_reference/input/examples/views/POST/ViewInput.json.example

```diff
@@ -0,0 +1,16 @@
+{
+    "ViewInput": {
+        "identifier": "TitleView",
+        "ContentQuery": {
+            "Filter": {
+                "ContentTypeIdentifierCriterion": "image",
+                "SectionIdentifierCriterion": "media"
+            },
+            "limit": 10,
+            "offset": 0,
+            "SortClauses": {
+                "ContentName": "ascending"
+            }
```

Comment:
Can we have the `contentTypeFacetBuilder` like [in the XML example](https://github.com/ezsystems/developer-documentation/blob/add-views-search-sample/docs/api/rest_api_reference/input/examples/views/POST/ViewInput.xml.example#L14)?